lib/client/order.rb

```diff
@@ -44,7 +58,14 @@ def get_shipping_addresses(order_id, params = {})
           response = client.get(uri({ order_id: order_id }, 'shipping_addresses'), params)
           return response.body
         rescue Faraday::Error => e
-          raise BigcommerceApiError.new('get order shipping address', { order_id: order_id }, e)
+          raise BigcommerceApiError.new(
+            e.response[:status],
+            'get order shipping address',
+            "Failed to get order shipping address #{order_id}",
+            order_id,
+            { order_id: order_id },
```

Comment:
Drop the `order_id` in the data object, and add the BC errors.

Reply:
@mrogers-5s resolved